plugins/GSdx/GSHwHack.cpp

```diff
@@ -629,23 +629,6 @@ bool GSC_TombRaiderUnderWorld(const GSFrameInfo& fi, int& skip)
 	return true;
 }
 
-bool GSC_FFVIIDoC(const GSFrameInfo& fi, int& skip)
```

Comment:
Are you sure OI_DoubleHalfClear also removed the garbage data that was shown when objects (dis)appear in the world?